docs/integrations/index.md

```diff
@@ -18,6 +18,7 @@ You must be assigned the tenant or organization **Owner** role to create, modify
 ## Related links:
 |        |               |               |
 | ------ | ------------- | ------------- |
+| Public IP Addresses | [Docs](https://turbot.com/pipes/docs/integrations/public-ip) | |
```

Comment:
Let's try moving this to `Reference` -> `Public IP Addresses`?

@johnsmyth thoughts?

Reply:
Agree  @MichaelBurgess  - I would put it in the reference section somewhere